bad_code.java

```diff
@@ -0,0 +1,54 @@
+public class BadCodeExample {
```

Comment:
It is a good practice to follow naming conventions for class names, method names, and variable names [important]